### ISSUE TITLE
Add 9 Lenovo A100 GPU nodes to OpenShift

### ISSUE THREAD

**Issue by joachimweyl** [nerc-project/operations#442]
## Motivation
OpenShift has current requests for GPUs. To meet these and the next few requests @waygil and @syockel decided to add 11 of the 15 nodes to OpenShift. 2 of them are not working so we changed this to 9 and will add the other 2 when they are available.

## Completion Criteria
9 nodes added to OpenShift and are accessible to be used.

## Question
Do we want to do the first testing in the OpenShift test cluster?

## Description
- [x] Add 9 GPU nodes to OpenShift Prod cluster
- [x] Create an issue to schedule a maintenance to make these live
- [x] Create an issue to add the 12th OpenShift node once the [16th Lenovo node is delivered](https://github.com/nerc-project/operations/issues/417)

## Completion dates
Desired - 2024-02-16
Required - 2024-02-28

**Comment by msdisme** [comment]
Also, we may need to be able to differentiate between different HW costs - eg. water cooled vs air cooled appears to have a different price.    Perhaps not for right away, but we should keep it in mind for when variations 
do get added.

**Comment by jtriley** [comment]
One of these hosts has recently hit a system board error in the event log. I can't power this host on anymore:
```
0	System	FQXSPPW0063M	Sensor SysBrd Vol Fault has transitioned to critical from a less severe state.
```
We might need to ship this host back to Lenovo.

**Comment by jtriley** [comment]
9/10 of the Lenovo nodes have been installed at this point in the OpenShift cluster. The other 2 (one with an OBM issue currently and the other waiting to be shipped from Lenovo) are slated to be added to the nerc-ocp-test cluster for testing MIG and other features.

**Comment by jtriley** [comment]
Unfortunately we discovered a networking issue that needs to be resolved and will require a rolling reboot of all the nodes in the cluster so we're removing these hosts for now and schedule a later maintenance date to add them back in given that we're out of time for this current planned maintenance.

**Comment by joachimweyl** [comment]
@jtriley how can we access the node that was added to the test cluster?

**Comment by jtriley** [comment]
Just noting the host (`wrk-90`) I mentioned above "with OBM issue waiting on techsquare support" has been fixed and added to the prod cluster.

**Comment by aabaris** [comment]
Made a separate issue to track repairs of wrk-99, linking it here.
https://github.com/nerc-project/operations/issues/492
